release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -24,8 +24,9 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 You must use {op-system} machines for the control plane, and you can use either {op-system} or {op-system-base} for compute machines.
 //Removed the note per https://issues.redhat.com/browse/GRPA-3517
 
-//TODO: Add this for 4.14
-Starting with {product-title} 4.12, an additional six months is added to the Extended Update Support (EUS) phase on even numbered releases from 18 months to two years. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
+Starting with {product-title} 4.12, Red Hat has added a six months Extended Update Support (EUS) phase for even numbered releases, extending the total available lifecycle from 18 months to two (2) years. In the {product-title} 4.12 release, this additional EUS is only available for the `x86_64` architecture.
+
+Starting with {product-title} 4.14, Red Hat has added an optional twelve months Additional Extended Update Support Term 2, extending the total available lifecycle from two (2) years to three (3) years. Additional EUS Term 2 is available on all architecture variants of {product-title}. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
```

Comment:
```suggestion
Starting with {product-title} 4.14, Red Hat has added an optional 12-month additional EUS Term 2, extending the total available lifecycle from 2 years to 3 years. The EUS Term 2 is available on all architecture variants of {product-title}. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
```

- I don't think you need to spell out EUS again since you did in the previous sentence.
- I noticed that in the previous sentence you use "EUS phase", while here it is called an "EUS Term 2". Are those two different things?

Reply:
Term 2 is a different level of support if customers want it.